server/src/main/java/io/druid/server/coordination/DruidServerMetadata.java

```diff
@@ -40,7 +43,10 @@ public DruidServerMetadata(
       @JsonProperty("maxSize") long maxSize,
       @JsonProperty("type") String type,
       @JsonProperty("tier") String tier,
-      @JsonProperty("priority") int priority
+      @JsonProperty("priority") int priority,
```

Comment:
I'm not convinced DruidServerMetadata should be continually added like this. There is likely a different set of metadata for different services.

I propose making the DruidServerMetadata a minimum set of static information about a service, rather than making it a catch-all for server metadata.